packages/terra-site/src/examples/form-checkbox/CheckboxFieldIndex.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-form-checkbox/docs/CheckboxField.md';
+import IndexTemplate from 'terra-dev-site/src/IndexPageTemplate';
```

Comment:
Re-iterating what @emilyrohrbough was saying - we can use `terra-doc-template` for these docs.